protocol/0044-LIME-lp_mechanics.md

```diff
@@ -374,3 +394,12 @@ In the case of spot markets it will be transferred into the network treasury for
   - `market.liquidityV2.sla.nonPerformanceBondPenaltySlope` valid values: `>=0`, `<=1000` default value of `2`  (<a name="0044-LIME-040" href="#0044-LIME-040">0044-LIME-040</a>)
   - `market.liquidityV2.sla.nonPerformanceBondPenaltyMax` valid values: `>=0`, `<=1` default value of `0.5`  (<a name="0044-LIME-041" href="#0044-LIME-041">0044-LIME-041</a>)
   - `market.liquidityV2.stakeToCcyVolume` valid values: `>=0`, `<=100` default value of `1`   (<a name="0044-LIME-042" href="#0044-LIME-042">0044-LIME-042</a>)
+
+#### Market parameters validation
+
+- Boundary values are respected for the market parameters
+  - `market.liquidity.providers.fee.calculationTimeStep` valid values: `>=0`, `<= validators.epoch.length` (<a name="0044-LIME-063" href="#0044-LIME-063">0044-LIME-063</a>)
```

Comment:
There is no upper boundary for this market parameter, should there be a sensible upper boundary value instead?
cc @karlem @davidsiska-vega

Reply:
will be addressed by pr https://github.com/vegaprotocol/vega/issues/9063

Reply:
what will be the default value for this new network parameter?

Reply:
1. with @karlem we agreed that this ought to be a network parameter and @karlem will provide the spec PR. 
2. the bounds are still `>=0` and `<=validators.epoch.length`. They need to be cross-validated against each other on change proposals. 
3. default value can be what `market.liquidity.providers.fee.distributionTimeStep` is set to now, which is `60m = 1h`.